lib/api/keys.ts

```diff
@@ -98,7 +134,19 @@ export const updatePrivateKeyRoute = ({ KeySalt, Keys, OrganizationKey }) => ({
     }
 });
 
-export const resetKeysRoute = ({ Username, PrimaryKey, Token, KeySalt, AddressKeys }) => ({
+export const resetKeysRoute = ({
+    Username,
+    PrimaryKey,
+    Token,
+    KeySalt,
+    AddressKeys
+}: {
+    Username: string;
+    PrimaryKey: string;
+    Token: string;
+    KeySalt: string;
+    AddressKeys: any[];
```

Comment:
Not sure how much you want to try hard on this type, but it comes from: `proton-shared/lib/keys/resetKeys.js` -> `getResetAddressesKeys()`

probably even `ReturnType<typeof getResetAddressesKeys>[]` could be done, not sure how correctly typescript infers types for that function.